packages/omuse-framework/setup.py

```diff
@@ -20,7 +20,7 @@
     'nose>=0.11.1',
     'mpi4py>=1.1.0',
     'h5py>=1.1.0',
-    'amuse-framework>=2021.3.1',
+    'amuse-devel>=2021.3.1',
```

Comment:
I would actually expect omuse-framework to use amuse framework. Seems a bit more stable than amuse-develop. What are we missing if we take amuse-framework? 
I can imagine that if you want to install omuse in development mode, you might want to also have amuse-devel. What do you think?